tests/integration/test_select.py

```diff
@@ -116,3 +116,11 @@ def test_timezones(c, datetime_table):
     result_df = result_df.compute()
 
     assert_frame_equal(result_df, datetime_table)
+
+
+def test_multi_case_when(c):
+    df = pd.DataFrame({"a": [1, 6, 7, 8, 9]})
+    c.create_table("df", df)
+
+    query = "SELECT CASE WHEN a BETWEEN 6 AND 8 THEN 1 ELSE 0 END FROM df"
+    c.sql(query).compute()
```

Comment:
Could we also check if this gives the expected result?

```suggestion
    actual_df = c.sql(
        """
    SELECT
        CASE WHEN a BETWEEN 6 AND 8 THEN 1 ELSE 0 END AS C
    FROM df
    """
    ).compute()
    expected_df = pd.DataFrame({"C": [0, 1, 1, 1, 0]})

    assert_frame_equal(actual_df, expected_df)
```